yarn.lock

```diff
@@ -14448,7 +14448,7 @@ react-fast-compare@^3.0.1:
   resolved "https://registry.yarnpkg.com/react-fast-compare/-/react-fast-compare-3.2.0.tgz#641a9da81b6a6320f270e89724fb45a0b39e43bb"
   integrity sha512-rtGImPZ0YyLrscKI9xTpV8psd6I8VAtjKCzQDlzyDvqJA8XOW78TXYQwNRNd8g8JZnDu8q9Fu/1v4HPAVwVdHA==
 
-react-hot-loader@^4.12.19, react-hot-loader@^4.3.11:
+react-hot-loader@^4.3.11:
```

Comment:
Why is there still a react-hot-loader here ? Is it wrongly declared in one of our dependency ?

Reply:
=> Found "react-hot-loader@4.12.21"
info Reasons this module exists
   - "cozy-ui" depends on it
   - Hoisted from "cozy-ui#react-hot-loader"
   - Hoisted from "cozy-bar#cozy-ui#react-hot-loader"

I guess the bar is not up to date with your latest change in UI (https://github.com/cozy/cozy-ui/commit/9e6f388df56209e4f6320225ab8c7082c90d75cf) 


(and thanks for to remind me I needed to check the lock file 👍 )

Reply:
after a clean of my lock file:

=> Found "react-hot-loader@4.13.0"
info Reasons this module exists
   - "cozy-bar#cozy-ui" depends on it
   - Hoisted from "cozy-bar#cozy-ui#react-hot-loader"

way better ;)